scripts/serve/serve.js

```diff
@@ -10,7 +10,7 @@ const webpackServe = (options) => {
   const serveConfig = {
     config,
     ...(port) && { port },
-    ...(host) && { host },
+    ...(host) && { host || '0.0.0.0' },
```

Comment:
This fallback will only be hit if called serve as a function. We we want to add a fallback for this case? We don't for the port value, which is defaulted to `8080` in the cli. If we do, would it be worth adding a server/serves-static defaults config file so we only need to update fallback values in one place?